jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/generator/WildflyJARGenerator.java

```diff
@@ -42,7 +40,7 @@ public class WildflyJARGenerator extends JavaExecGenerator {
     public WildflyJARGenerator(GeneratorContext context) {
         super(context, "wildfly-jar");
         JavaProject project = context.getProject();
-        Plugin plugin = JKubeProjectUtil.getPlugin(project, BOOTABLE_JAR_GROUP_ID, BOOTABLE_JAR_ARTIFACT_ID);
+        Plugin plugin = JKubeProjectUtil.getPlugin(project, "org.wildfly.plugins", "wildfly-jar-maven-plugin");
```

Comment:
I'm not sure if you interpreted this from a comment in one of the tests.

**Production code** should be using the constants, that's not a problem.

However, the test code SHOULD NOT use the constants, especially if they are the same as the ones in the production code. The reason is simple, a change in the constant, will not make the test fail even though we've completely changed the specification constraint. Considering a test is used to verify a specification or a defined behavior, we should use either a literal, or if needed a variable defined in the same test (however, this might make the test less readable).

Reply:
Got it